changelog.rst

```diff
@@ -78,6 +78,24 @@ Breaking Changes
   Make sure to use the full path now when using client methods. (:pr:`1185`)
 
 - Updated MappedCollectionDoc and GuestCollectionDoc with MissingType. (:pr:`1189`)
+.. _changelog-3.60.0:
```

Comment:
Suggesting a single blank line here because I harbor concerns that this may render incorrectly in some contexts.

```suggestion

.. _changelog-3.60.0:
```